client/extensions/woocommerce/index.js

```diff
@@ -84,6 +86,18 @@ const getStorePages = () => {
 			path: '/store/promotions/:site',
 		},
 		{
+			container: PromotionCreate,
+			configKey: 'woocommerce/extension-promotions',
+			documentTitle: translate( 'New Promotion' ),
```

Comment:
Hi! I've found a possible matching string that has already been translated **27** times:
`translate( 'Promotions' )` *ES Score: 8*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).